endercore/src/main/java/com/enderio/core/common/blockentity/sync/EnderDataSlot.java

```diff
@@ -0,0 +1,46 @@
+package com.enderio.core.common.blockentity.sync;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.Accessors;
+import net.minecraft.nbt.CompoundTag;
+
+import java.util.Optional;
+import java.util.function.Consumer;
+import java.util.function.Supplier;
+
+@RequiredArgsConstructor
```

Comment:
lombok agan. Personally I feel like while it may result in writing less, people would have to learn what all of these annotations mean. I'd rather sticking to vanilla Java.

Reply:
Lombok is still present in EnderCore, is this intentional? Don't worry if not, just performing final review before I'm happy to merge :)